docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -361,9 +385,22 @@ This time we will look for the ``StructureData`` node (green circle) that has an
 
      The `details` view of the ``StructureData`` node that corresponds to the original ``BandsData`` node.
 
-We can see in this case that the original ``BandsData`` corresponds to a Silica structure.
+We can see in this particular case that the original ``BandsData`` corresponds to a Silica structure (your final structure might be different).
 You can look at the structure here, explore the details of the cell, etc.
 
+**Exercise:**
+Once your workflow has finished running, identify the band structure that was produced by it (probably the one with the newest creation time).
+Repeat the same procedure to find out the relaxed structure that was used for this calculation.
+Now, as we explained above, the workflow has a relaxation step, so you could look at the original crystal structure to see if the geometry changed during relaxation.
+To do so:
+
+    1. Go to the details page of the ``BandsData`` node.
+    2. Look in the Provenance Browser for the workflow node that corresponds to the outermost ``PwBandStructureWorkChain``.
+       You identify it by hovering over the nodes and checking that the UUID corresponds with the one in your database.
+       Once you click on it and see its details, verify that it is in effect the ``PwBandStructureWorkChain``
+    3. Select the input structure of that workflow node.
+       It is probably hard to identify differences since the changes in vector cells and atomic positions might be just due to rotations, but the relevant questions are: has the distance between atoms changed? has the total volumen changed?
```

Comment:
```suggestion
       It is probably hard to visually identify differences since the changes in vector cells and atomic positions might be just due to rotations, but the relevant questions are: has the distance between atoms changed? has the total volume changed?
```